docs/source/sphinxext/comment_eater.py

```diff
@@ -1,10 +1,11 @@
-from cStringIO import StringIO
```

Comment:
I think some of the bundled sphinx extensions in this folder might not be Python3 compatible... (Just a comment in passing, no need to take any action)